scripts/e2e.ganache.sh

```diff
@@ -26,7 +26,8 @@ echo " "
 npx ganache-cli --noVMErrorsOnRPCResponse > /dev/null &
 client=$!
 npx wait-port 8545
-
+ls -la /home/runner/work/web3.js/web3.js/packages/web3/
+ls -la /home/runner/work/web3.js/web3.js/packages/web3/lib
```

Comment:
should this still be here? i assume it was for some debugging

Reply:
:facepalm: woops